From a2be69d47884dd995ac6e9004ba2855f354f7522 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <jan.steffens@gmail.com>
Date: Sun, 15 Dec 2019 00:07:08 +0100
Subject: Client: Fix detection of linux-dmabuf

Change I84c8c1008724b49b6bedb4fc3ef398e292f1c6c7 fixed the tests in
compositor/configure.json but missed the test in client/configure.json.

Change-Id: I65ad424406438baa74ca80a9418e133510142118
Reviewed-by: Johan Helsing <johan.helsing@qt.io>
---
 src/client/configure.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/client/configure.json b/src/client/configure.json
index e9e16324..06213968 100644
--- a/src/client/configure.json
+++ b/src/client/configure.json
@@ -74,7 +74,7 @@
             "label": "Linux dma-buf Buffer Sharing",
             "type": "compile",
             "test": "dmabuf_server_buffer",
-            "use": "egl"
+            "use": "egl drm"
         },
         "vulkan-server-buffer": {
             "label": "Vulkan Buffer Sharing",
-- 
cgit v1.2.1

